libmscore/chord.cpp

```diff
@@ -2523,11 +2530,11 @@ Element* Chord::drop(EditData& data)
 
             case ElementType::TREMOLO:
                   {
-                  Tremolo* t = toTremolo(e);
+                  Tremolo* t = static_cast<Tremolo*>(e);
```

Comment:
Please keep MuseScore 3 syntax and use toXXXX converter.
See https://github.com/musescore/MuseScore/blob/master/mscore3.txt#L116